docs/_includes/generated/yarn_config_configuration.html

```diff
@@ -52,6 +52,11 @@
             <td style="word-wrap: break-word;">5</td>
             <td>Time between heartbeats with the ResourceManager in seconds.</td>
         </tr>
+        <tr>
+            <td><h5>yarn.max-container-launcher-number</h5></td>
+            <td style="word-wrap: break-word;">10</td>
+            <td>The max number of threads for starting yarn containers in yarn resource manager.</td>
```

Comment:
technically, this argument allows user to set the "EXACT" num of thread for strarting, instead of the maximum.